src/drive/targets/mobile/README.md

```diff
@@ -4,16 +4,16 @@
 
 ### :wrench: Requirements
 
-- Node v10
-  - on macOS: `brew install node@10 && brew link node@10`
+- Node 14
+  - on macOS: `brew install node@14 && brew link node@14`
 - ImageMagick
   - on macOS: `brew install imagemagick`
-- Cordova v7 CLI
-  - `yarn add cordova@7 -g`
+- Cordova 8.1.2 CLI - [documentation here](https://cordova.apache.org/docs/en/7.x/reference/cordova-cli/index.html)
```

Comment:
the link is not good.

https://cordova.apache.org/docs/en/8.x/reference/cordova-cli/index.html

Reply:
bien vu !